index.html

```diff
@@ -19,7 +20,16 @@
             <div class="Navbar">
                 <a href="#"><img src="req/mylogo.png" alt="logo" id="mylogo"></a>
                 <!--Nav bar button-->
-                <img id="logo" src="req/Union.png" alt="logo">
+                <button class="hamburger">
+                    <i class="menuIcon material-icons">menu</i>
+                    <i class="closeIcon material-icons">close</i>
+                  </button>
+                
+                <ul class="menu">
+                    <li><a class="menuItem" href="#works">Portfolio</a></li>
+                    <li><a class="menuItem" href="#about-section">About</a></li>
+                    <li><a class="menuItem" href="#Contact">Contact</a></li>
```

Comment:
- [ ] there's a small bug that should be fixed to keep your menu work correctly, whenever i open the menu and scroll down while it's open the X button will disappear and I won't be able to close it anymore, i suggest that you remove the ability to scroll whenever the menu is open , kindly fix it 🙏 

![closebuggggggg](https://user-images.githubusercontent.com/89282221/223391986-75641b62-9e09-4d65-8a09-53540baf0a86.gif)